scripts/bootstrap.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env babel-node
+
+import inquirer from 'inquirer';
+import program from 'commander';
+import childProcess from 'child_process';
+
+const spawn = childProcess.spawnSync;
+
+program
+  .version('1.0.0')
+  .option('--all', 'Run all without asking')
```

Comment:
I'm assuming developers want to type LESS ?

I think interactive mode is a sane default for a developer tool ?